torch/backends/__init__.py

```diff
@@ -64,6 +64,7 @@ def __getattr__(self, attr):
     mkldnn as mkldnn,
     mps as mps,
     nnpack as nnpack,
+    onednn as onednn,
```

Comment:
I'm not sure if we need to add `onednn` or reuse `mkldnn`. @jgong5 , any thoughts?

Reply:
Better to use mkldnn for consistency.

Reply:
@weishi-deng , please use `mkldnn` first. In the future, we can add `onednn` if we think it is the right time to do so then.

Reply:
Yes, will do.